tests/posix/fs/src/test_fs_file.c

```diff
@@ -90,15 +90,41 @@ static int test_file_read(void)
 
 	read_buff[brw] = 0;
 
-	TC_PRINT("Data read:\"%s\"\n\n", read_buff);
+	TC_PRINT("Data read:\"%s\"\n", read_buff);
 
 	if (strcmp(test_str, read_buff)) {
 		TC_PRINT("Error - Data read does not match data written\n");
 		TC_PRINT("Data read:\"%s\"\n\n", read_buff);
 		return TC_FAIL;
 	}
 
-	TC_PRINT("Data read matches data written\n");
+	/* Now test after non-zero lseek. */
+
+	res = lseek(file, 2, SEEK_SET);
+	if (res != 0) {
+		TC_PRINT("lseek failed [%d]\n", (int)res);
+		close(file);
+		return TC_FAIL;
+	}
+
+	brw = read(file, read_buff, sizeof(read_buff));
+	if (brw < 0) {
+		TC_PRINT("Failed reading file [%d]\n", (int)brw);
+		close(file);
+		return TC_FAIL;
+	}
+
+	read_buff[brw] = 0;
+
+	TC_PRINT("Data read:\"%s\"\n", read_buff);
+
+	if (strcmp(test_str + 2, read_buff)) {
+		TC_PRINT("Error - Data read does not match data written\n");
+		TC_PRINT("Data read:\"%s\"\n\n", read_buff);
```

Comment:
Double empty line - not sure if it was intended.

Reply:
@rlubos, you mean in the TC_PRINT output? That's just copy-paste from existing code, which thus does that. If fixed, should be done consistently, and patching test output of this test is completely unrelated to the state description of the patch/PR. (I actually gave in, and patched one case, if you look above in this file - but I verified that with it the output is guaranteedly looks better, and without it - worse).

Reply:
It only drew my attention beacuse you actually fix similar case a few lines above.
But really, it's such a big nit that I don't see a reason to elaborate/argue on that.

Reply:
> you actually fix similar case a few lines above.

Just one of a dozen.

> But really, it's such a big nit that I don't see a reason to elaborate/argue on that.

Yeah, tests are tests - it's good that someone writes them and elaborates at all ;-).